<ui version="4.0" >
 <class>ICatAdvancedSearch</class>
 <widget class="QWidget" name="ICatAdvancedSearch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ICat Advanced Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="3" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>568</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>221</height>
      </size>
     </property>
     <property name="styleSheet" >
      <string notr="true" >background-color: rgb(216, 225, 255);</string>
     </property>
     <column>
      <property name="text" >
       <string>                                                                                                       Advanced Investigation Search</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QTableWidget" name="advSearchtableWidget" >
     <property name="styleSheet" >
      <string notr="true" >background-color: rgb(216, 225, 255);</string>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="advsearchLabel" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QLineEdit" name="startdateLineEdit" >
   <property name="geometry" >
    <rect>
     <x>411</x>
     <y>67</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>101</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTip" >
    <string>Date in DD/MM/YYY format</string>
   </property>
   <property name="inputMask" >
    <string>99/99/9999;_</string>
   </property>
   <property name="text" >
    <string>//</string>
   </property>
  </widget>
  <widget class="QLabel" name="startdateLabel" >
   <property name="geometry" >
    <rect>
     <x>301</x>
     <y>69</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Start Date:</string>
   </property>
   <property name="buddy" >
    <cstring>startdateLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="runNumberLabel" >
   <property name="geometry" >
    <rect>
     <x>301</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Run#:</string>
   </property>
   <property name="buddy" >
    <cstring>startRunEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="keywordsLabel" >
   <property name="geometry" >
    <rect>
     <x>532</x>
     <y>128</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>KeyWord(s):</string>
   </property>
   <property name="buddy" >
    <cstring>keywordsEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="enddateLabel" >
   <property name="geometry" >
    <rect>
     <x>301</x>
     <y>93</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>End Date:</string>
   </property>
   <property name="buddy" >
    <cstring>enddateLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="enddateLineEdit" >
   <property name="geometry" >
    <rect>
     <x>411</x>
     <y>93</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>101</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTip" >
    <string>Date in DD/MM/YYY format</string>
   </property>
   <property name="inputMask" >
    <string>99/99/9999;_</string>
   </property>
   <property name="text" >
    <string>//</string>
   </property>
  </widget>
  <widget class="QComboBox" name="instrumentBox" >
   <property name="geometry" >
    <rect>
     <x>411</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>101</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTip" >
    <string>Instrument Box</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startRunEdit" >
   <property name="geometry" >
    <rect>
     <x>411</x>
     <y>41</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>81</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTip" >
    <string>Start Run Number</string>
   </property>
   <property name="inputMask" >
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="enddatetoolButton" >
   <property name="geometry" >
    <rect>
     <x>532</x>
     <y>90</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>25</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="endRunEdit" >
   <property name="geometry" >
    <rect>
     <x>532</x>
     <y>41</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>81</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTip" >
    <string>End Run Number</string>
   </property>
   <property name="inputMask" >
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="startdatetoolButton" >
   <property name="geometry" >
    <rect>
     <x>532</x>
     <y>66</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>25</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keywordsEdit" >
   <property name="geometry" >
    <rect>
     <x>605</x>
     <y>128</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>121</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton" >
   <property name="geometry" >
    <rect>
     <x>660</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Search Investigations</string>
   </property>
   <property name="text" >
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="instrumentLabel" >
   <property name="geometry" >
    <rect>
     <x>301</x>
     <y>132</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Instrument:</string>
   </property>
   <property name="buddy" >
    <cstring>instrumentBox</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="investigatonNameEdit" >
   <property name="geometry" >
    <rect>
     <x>137</x>
     <y>41</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>133</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>41</y>
     <width>112</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Investigation Name:</string>
   </property>
   <property name="buddy" >
    <cstring>investigatonNameEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>67</y>
     <width>112</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Investigation Abstract:</string>
   </property>
   <property name="buddy" >
    <cstring>invstAbstractEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="invstAbstractEdit" >
   <property name="geometry" >
    <rect>
     <x>137</x>
     <y>67</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>133</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>93</y>
     <width>112</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Sample:</string>
   </property>
   <property name="buddy" >
    <cstring>sampleEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="sampleEdit" >
   <property name="geometry" >
    <rect>
     <x>137</x>
     <y>93</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>133</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>119</y>
     <width>112</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Investigators surname:</string>
   </property>
   <property name="buddy" >
    <cstring>invstsurnameEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="invstsurnameEdit" >
   <property name="geometry" >
    <rect>
     <x>137</x>
     <y>119</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>133</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>145</y>
     <width>112</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Datafile name:</string>
   </property>
   <property name="buddy" >
    <cstring>datafilenameEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="datafilenameEdit" >
   <property name="geometry" >
    <rect>
     <x>137</x>
     <y>145</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>133</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="casesensitiveBox" >
   <property name="geometry" >
    <rect>
     <x>136</x>
     <y>173</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Case Sensitive</string>
   </property>
  </widget>
  <widget class="QLabel" name="invstTypeLabel" >
   <property name="geometry" >
    <rect>
     <x>301</x>
     <y>171</y>
     <width>93</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Investigation Type:</string>
   </property>
   <property name="buddy" >
    <cstring>invstTypeBox</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="invstTypeBox" >
   <property name="geometry" >
    <rect>
     <x>411</x>
     <y>170</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>101</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <zorder>treeWidget</zorder>
  <zorder>startdateLineEdit</zorder>
  <zorder>startdateLabel</zorder>
  <zorder>runNumberLabel</zorder>
  <zorder>keywordsLabel</zorder>
  <zorder>enddateLabel</zorder>
  <zorder>enddateLineEdit</zorder>
  <zorder>instrumentBox</zorder>
  <zorder>startRunEdit</zorder>
  <zorder>enddatetoolButton</zorder>
  <zorder>endRunEdit</zorder>
  <zorder>startdatetoolButton</zorder>
  <zorder>keywordsEdit</zorder>
  <zorder>searchButton</zorder>
  <zorder>instrumentLabel</zorder>
  <zorder>investigatonNameEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>invstAbstractEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>sampleEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>invstsurnameEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>datafilenameEdit</zorder>
  <zorder>casesensitiveBox</zorder>
  <zorder>invstTypeLabel</zorder>
  <zorder>invstTypeBox</zorder>
  <zorder>advSearchtableWidget</zorder>
  <zorder>advsearchLabel</zorder>
 </widget>
 <tabstops>
  <tabstop>investigatonNameEdit</tabstop>
  <tabstop>startRunEdit</tabstop>
  <tabstop>endRunEdit</tabstop>
  <tabstop>invstAbstractEdit</tabstop>
  <tabstop>startdateLineEdit</tabstop>
  <tabstop>startdatetoolButton</tabstop>
  <tabstop>sampleEdit</tabstop>
  <tabstop>enddateLineEdit</tabstop>
  <tabstop>enddatetoolButton</tabstop>
  <tabstop>invstsurnameEdit</tabstop>
  <tabstop>instrumentBox</tabstop>
  <tabstop>keywordsEdit</tabstop>
  <tabstop>datafilenameEdit</tabstop>
  <tabstop>casesensitiveBox</tabstop>
  <tabstop>invstTypeBox</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>advSearchtableWidget</tabstop>
  <tabstop>treeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
